backend/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": ".venv\\Scripts\\python.exe"
```

Comment:
This is Windows specific. I'd not commit it and keep it in the /Users/ settings of your VSCode (not in the project settings).